ci/ansible/roles/pulp-smash/defaults/main.yml

```diff
@@ -9,3 +9,9 @@ pulp_smash_password: "admin"
 
 # If SSL certificate should be verified
 pulp_smash_verify: "false"
+
+# Pulp Server version
+# pulp_smash_version variable is optional, if you want to specify it then you
+# can do so by passing additional variables to ansible-playbook command or by
+# uncommenting the following line and editing as needed.
+# pulp_smash_version: 2.7
```

Comment:
@seandst grabbing that from a config file will even help testing if the Pulp Server is reporting the right version. This was a design decision within Pulp Smash I'm just replicating it to the play book that runs it.

Also regarding the default with comment this is something optional and Pulp Smash will consider latest Pulp Server if not version is provided. I left the comment since this is the first place I would check for expected variables on a playbook role. Worth mention that I'm learning Ansible and it could be a better place to document it.